plottable.d.ts

```diff
@@ -2171,6 +2171,7 @@ declare module Plottable {
     }
 }
 
+
```

Comment:
This is a bit weird. Could you double check that the compilation actually yielded that?